client/ayon_core/modules/timers_manager/rest_api.py

```diff
@@ -71,11 +71,11 @@ async def get_task_time(self, request):
         data = await request.json()
         try:
             project_name = data['project_name']
-            asset_name = data['asset_name']
+            asset_name = data['folder_path']
```

Comment:
I've said this before elsewhere - but is there a reason we're not refactoring the variable names?

Reply:
There are multiple opened big PRs changing the same lines. It's PR conflicts hell why it's not changed.